src/Integration.Vsix/CFamily/CFamilyStrings.resx

```diff
@@ -123,6 +123,9 @@
   <data name="ERROR_ProcessRunner_ExeNotFound" xml:space="preserve">
     <value>Execution failed. The specified executable does not exist: {0}</value>
   </data>
+  <data name="ERROR_ProcessRunner_Failed" xml:space="preserve">
+    <value>Process failed, error code: {0}</value>
```

Comment:
`Process failed. Exit code:`

... might be better (both grammatically, and because it's normal to talk about exit codes for processes)